src/repositories/adRepository.ts

```diff
@@ -0,0 +1,10 @@
+import { Ad, IAdModel } from '../db/';
+
+export const save = async (ad: object) => {
+  const newAd = new Ad(ad);
+  newAd.save(err => err);
```

Comment:
What is it and how this is going to work? err => err won't throw anything.

Reply:
fixed